srcalc/src/main/java/gov/va/med/srcalc/web/view/admin/EditRule.java

```diff
@@ -0,0 +1,155 @@
+package gov.va.med.srcalc.web.view.admin;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+
+import gov.va.med.srcalc.domain.model.Rule;
+import gov.va.med.srcalc.domain.model.ValueMatcher;
+import gov.va.med.srcalc.domain.model.Variable;
+import gov.va.med.srcalc.service.AdminService;
+import gov.va.med.srcalc.service.InvalidIdentifierException;
+
+public class EditRule
+{
+    private AdminService fAdminService;
+    private List<ValueMatcherBuilder> fMatchers;
+    private String fSummandExpression;
+    private boolean fRequired;
+    private String fDisplayName;
+    private String fNewVariableKey;
+    
+    public EditRule(final AdminService adminService) 
```

Comment:
I think this can be just a `ModelInspectionService` instead of an `AdminService`. Be as flexible as possible in what you accept.